docs/02-app/01-building-your-application/03-rendering/01-server-components.mdx

```diff
@@ -131,6 +131,6 @@ With Streaming, routes are rendered on the server at **request time**. The work
   height="785"
 />
 
-Streaming is useful for lower-priority UI, or UI that depends on slower data fetches that would block rendering for the whole route. For example, reviews on a product page.
+Streaming is built into the Next.js App Router by default. This helps improve both the initial page loading performance, as well as UI that depends on slower data fetches that would block rendering the whole route. For example, reviews on a product page.
 
-In Next.js, you can stream route segments using `loading.js`, and UI components with [React Suspense](https://react.dev/reference/react/Suspense). See the [Loading UI and Streaming](/docs/app/building-your-application/routing/loading-ui-and-streaming) docs for more information.
```

Comment:
double checking: did we mean to update the React Suspense link to go to the loading docs?

Reply:
Yes, because it's then linked back in those docs. I want people to read through that page first 👍